app/src/test/java/com/example/habitsmasher/ui/dashboard/HabitListTest.java

```diff
@@ -37,14 +37,14 @@ public void addHabit_validHabitAddition_expectHabitAddedToList(){
     public void editHabit_validEdit_expectHabitToBeEdited() {
         long habitId = 0;
 
-        Habit habit = new Habit("Title 1", "Reason 1", new Date(), "MO", habitId, EMPTY_HABIT_EVENTS_LIST);
+        Habit habit = new Habit("Title 1", "Reason 1", new Date(), "MO", true,  habitId, EMPTY_HABIT_EVENTS_LIST);
```

Comment:
Can we extract this out to a constant in this test class so it's **descriptive**, and not just an isolated boolean? Something like `Boolean PUBLIC_HABIT = true`.
- [ ] extract boolean to constant